File: components/auth/src/main/java/com/ecfront/dew/auth/AuthApplication.java
package com.ecfront.dew.auth;

import com.ecfront.dew.core.Dew;
import com.ecfront.dew.core.DewCloudApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.context.annotation.ComponentScan;

@ComponentScan(basePackageClasses = {Dew.class,AuthApplication.class})
public class AuthApplication extends DewCloudApplication {

    public static void main(String[] args) {
        new SpringApplicationBuilder(AuthApplication.class).web(true).run(args);
    }

}

File: components/auth/src/main/scala/com/tairanchina/csp/dew/auth/AuthApplication.scala
package com.tairanchina.csp.dew.auth

import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.cloud.client.SpringCloudApplication
import org.springframework.transaction.annotation.EnableTransactionManagement

@SpringCloudApplication
@EnableTransactionManagement
class AuthApplication {

}

object AuthApplication extends App {

  new SpringApplicationBuilder(classOf[AuthApplication]).web(true).run(args: _*)

}
